tests/console/suse_module_tools.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Testing functionality of modhash and modsign-verify commands.
+# A random module from available loadable modules on the system is picked
+# for testing purposes.
+# * modhash: the output is checked for correct format
+# * modsign-verify: the output is checked for correct format
+# Maintainer: Vasileios Anastasiadis <vasilios.anastasiadis@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    # Find a random module available on the system
+    # Get a list of all modules
+    my $mds = script_output("find /lib/modules/\$(uname -r) -type f -name '*.ko'");
+    # Get the first module on the list
+    my $pth = (split '\n', $mds)[0];
+
+    # Test modhash command
+    if (!is_sle('=12-sp1') && !is_sle('=12-sp5')) {
+        # Get the output
+        my $hash = script_output("modhash $pth");
```

Comment:
```suggestion
assert_script_run "modhash $pth | grep '[0-9a-fA-F]'";
```

Reply:
I will apply that in the morning! It is reasonable. Thanks!

Reply:
I applied your suggestion and tested. Everything works as expected. I mark this as resolved.